blog/en/blog/2025/06/30/2025-june-monthly-report.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "2025 Monthly Report (June 01 - June 30)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.api7.ai/uploads/2025/06/30/hNrgfyse_june-monthly-report-cover-en.webp
+---
+
+> Recently, we've introduced and updated some new features, including adding devcontainer support, enhancing Admin API filtering, and adding `headers` attribute for `loki-logger` plugin, etc. For more details, please read this month's newsletter.
+
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way and is made possible by the collective effort of our community.
+
+From June 1st to June 30, 17 contributors made 86 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.api7.ai/uploads/2025/06/30/ctN7FHKp_june-contributor-list.webp)
+
+![New Contributors List](https://static.api7.ai/uploads/2025/06/30/VDMkGEcj_june-new-contributors.webp)
+
+## Feature Highlights
+
+### 1. Add devcontainer support
+
+PR: https://github.com/apache/apisix/pull/11765
+
+Contributor: [bzp2010](https://github.com/bzp2010)
+
+This PR adds devcontainer support for Linux, Windows (WSL2), and macOS across amd64/arm64 architectures, enabling seamless integration with mainstream development environments and ready-to-use functionality. Additionally, etcd is maintained using docker-compose and accessible on the local loopback.
+
+### 2. Add `headers` attribute for `loki-logger` plugin
+
+PR: https://github.com/apache/apisix/pull/11420
+
+Contributor: [slow-groovin](https://github.com/slow-groovin)
+
+This PR adds a new headers attribute `authorization` for the `loki-logger` plugin to provide an HTTP authorization header when using non-local Loki services.
```

Comment:
The info is inaccurate in the PR description. Please refer to release notes for the description of this PR.

Reply:
fixed